### PR TITLE
WSL finetuning

### PR DESCRIPTION
**Pull request by jensens**
WSL chapter:
- remove dup section 
- minor ReST fixes
- finetuning in formatting

Windows section: mention and link WSL chapter

### CODE REVIEW

File: manage/installing/installation.rst [hunk 1]
```diff
@@ -310,6 +310,9 @@ For earlier versions of Plone 5, you have the `Windows installer <https://github
 
 For the Plone 4.3 series, there is a `binary installer <https://plone.org/products/plone/releases/4.3.6>`_.
 
+An alternative to directly running Plone 5 on Winodws is to use the official Windows Subsystem Linux (WSL, formerly also known as LXSS).
```
Comment (idgserpro) [inline manage/installing/installation.rst]: There's a small typo here, `Winodws` instead of `Windows`.

Comment (stevepiercy) [inline manage/installing/installation.rst]:
```suggestion
An alternative to directly running Plone 5 on Windows is to use the official Windows Subsystem Linux (WSL, formerly also known as LXSS).
```

File: manage/installing/installation_windows10_wsl.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ Pre Settings
 ------------
 
 Before installing the shell to later configure in, the first thing to do is to enable the optional feature for a subsystem on windows.
-Therefor, the programm Windows PowerShell has to be launched (as administrator! - right-click the application and choose "launch as admin").
+Therefor, the programm Windows PowerShell has to be launched (as administrator! - right-click the application and choose :guilabel:`launch as admin`).
```
Comment (stevepiercy) [inline manage/installing/installation_windows10_wsl.rst]:
```suggestion
Therefor, the program Windows PowerShell has to be launched (as administrator! - right-click the application and choose :guilabel:`launch as admin`).
```

File: manage/installing/installation_windows10_wsl.rst [hunk 3]
```diff
@@ -203,10 +170,11 @@ Before creating an add-on, the correct path has to be chosen.
 To edit the code via Windows and execute it under Ubuntu with Plone CLI, a shared place accessible from both systems is needed.
 
 Under Ubuntu this location is ``/mnt/c/`` for the Windows C-drive.
-I.e. create a folder in Windows ``C:\Plone-Projects``. In Linux it is located under ``/mnt/c/Plone-Projects``.
+I.e. create a folder in Windows ``C:\Plone-Projects``.
```
Comment (stevepiercy) [inline manage/installing/installation_windows10_wsl.rst]:
```suggestion
Create a folder in Windows ``C:\Plone-Projects``.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor typo and grammar fixes.

File: manage/installing/installation_windows10_wsl.rst [hunk 4]
```diff
@@ -18,7 +18,7 @@ Pre Settings
 ------------
 
 Before installing the shell to later configure in, the first thing to do is to enable the optional feature for a subsystem on windows.
-Therefor, the programm Windows PowerShell has to be launched (as administrator! - right-click the application and choose "launch as admin").
+Therefor, the program Windows PowerShell has to be launched (as administrator! - right-click the application and choose :guilabel:`launch as admin`).
```
Comment (stevepiercy) [inline manage/installing/installation_windows10_wsl.rst]:
```suggestion
Therefore the program Windows PowerShell has to be launched (as administrator! - right-click the application and choose :guilabel:`launch as admin`).
```

File: manage/installing/installation_windows10_wsl.rst [hunk 5]
```diff
@@ -18,7 +18,7 @@ Pre Settings
 ------------
 
 Before installing the shell to later configure in, the first thing to do is to enable the optional feature for a subsystem on windows.
```
Comment (stevepiercy) [inline manage/installing/installation_windows10_wsl.rst]:
```suggestion
Before installing the shell for configuration, first enable the optional feature for a subsystem on Windows.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Thanks for the changes. I found a couple more upon more careful review.

**Pull request merged**